inputFiles/thermalSinglePhaseFlowFractures/fractureMatrixThermalFlow_edfm_smoke.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" ?>
+
+<Problem>
+  <Included>
+    <File name="./fractureMatrixThermalFlow_edfm_base.xml" />
+    <File name="./fractureMatrixThermalFlow_bc.xml" />
+  </Included>
+
+  <Mesh>
+    <InternalMesh
+      name="mesh1"
+      elementTypes="{ C3D8 }"
+      xCoords="{ 0, 20 }"
+      yCoords="{ 0, 20 }"
+      zCoords="{ 0, 1 }"
+      nx="{ 11 }"
+      ny="{ 10 }"
+      nz="{ 1 }"
+      cellBlockNames="{ cb1 }"/>
+  </Mesh>
```

Comment:
this mesh is idem to benchmark case, do you want to reduce nx, ny?

Reply:
I d say that eventually we will want he benchmark one to be finer.